hack/local-up-volcano.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# Copyright 2019 The Volcano Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VK_ROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/..
+CLUSTER_NAME=${CLUSTER_NAME:-volcano}
+CLUSTER_CONTEXT="--name ${CLUSTER_NAME}"
+KIND_OPT=${KIND_OPT:-}
+INSTALL_MODE=${INSTALL_MODE:-"kind"}
+IMAGE_PREFIX=volcanosh/vc
+TAG=${TAG:-`git rev-parse --verify HEAD`}
+RELEASE_DIR=_output/release
+RELEASE_FOLDER=${VK_ROOT}/${RELEASE_DIR}
+YAML_FILENAME=volcano-${TAG}.yaml
+
+
+# check if kind installed
+function check-prerequisites {
+  echo "checking prerequisites"
+  which kubectl >/dev/null 2>&1
+  if [[ $? -ne 0 ]]; then
+    echo "kubectl not installed, exiting."
+    exit 1
+  else
+    echo -n "found kubectl, " && kubectl version --short --client
+  fi
+}
+
+# check if kind installed
+function check-kind {
+  echo "checking kind"
+  which kind >/dev/null 2>&1
+  if [[ $? -ne 0 ]]; then
+    echo "kind not installed, exiting."
+    exit 1
+  else
+    echo -n "found kind, version: " && kind version
+  fi
+}
+
+# prepare deploy yaml and docker images
+function prepare {
+  echo "Preparing..."
+  echo "Generating valcano deploy yaml"
+  make generate-yaml
+
+  echo "Building docker images"
+  make images
+}
+
+# spin up cluster with kind command
+function kind-up-cluster {
+  check-kind
+  echo "Running kind: [kind create cluster ${CLUSTER_CONTEXT} ${KIND_OPT}]"
+  kind create cluster ${CLUSTER_CONTEXT} ${KIND_OPT}
+
+  echo "Loading docker images into kind cluster"
+  kind load docker-image ${IMAGE_PREFIX}-controllers:${TAG}  ${CLUSTER_CONTEXT}
+  echo kind load docker-image ${IMAGE_PREFIX}-controllers:${TAG}  ${CLUSTER_CONTEXT}
+  kind load docker-image ${IMAGE_PREFIX}-scheduler:${TAG}  ${CLUSTER_CONTEXT}
+  kind load docker-image ${IMAGE_PREFIX}-admission:${TAG}  ${CLUSTER_CONTEXT}
+}
+
+function install-volcano {
+  # TODO: add a graceful way waiting for all crd ready
+  kubectl apply -f ${RELEASE_FOLDER}/${YAML_FILENAME} --kubeconfig ${KUBECONFIG}
+  sleep 5
+  kubectl apply -f ${VK_ROOT}/installer/helm/chart/volcano/templates/default-queue.yaml --kubeconfig ${KUBECONFIG}
```

Comment:
just have a retry should be enough :)

Reply:
I want to make the default-queue install into the scheduler if possible.